README.md

```diff
@@ -1,129 +1,114 @@
 # SHARK
 
-High Performance Machine Learning Distribution
+ SHARK is a high performarce machine learning distribution that is built on LLVM, MLIR, OpenXLA’s IREE and nod.ai’s tuning.
 
 [![Nightly Release](https://github.com/nod-ai/SHARK/actions/workflows/nightly.yml/badge.svg)](https://github.com/nod-ai/SHARK/actions/workflows/nightly.yml)
 [![Validate torch-models on Shark Runtime](https://github.com/nod-ai/SHARK/actions/workflows/test-models.yml/badge.svg)](https://github.com/nod-ai/SHARK/actions/workflows/test-models.yml)
 
+ 
+## Windows 10/11 QuickStart Guide
 
-<details>
-  <summary>Prerequisites - Drivers </summary>
-  
-#### Install your Windows hardware drivers
+### Installation
+
+Install Windows Hardware Drivers
 * [AMD RDNA Users] Download the latest driver [here](https://www.amd.com/en/support/kb/release-notes/rn-rad-win-23-2-1).
```

Comment:
Now that MLIR/IREE support is integrated into the mainline drivers, I think its a bit problematic to link to a particular driver version as the 'latest', as we're not set up to keep updating to link to be to the actual latest one.

Maybe we should just say > 23.2.1 and link to the main AMD support page at https://www.amd.com/en/support